libbeat/cfgfile/registry.go

```diff
@@ -1,39 +1,39 @@
-package module
+package cfgfile
 
 import "sync"
 
-type registry struct {
+type Registry struct {
 	sync.Mutex
 	List map[uint64]Runner
```

Comment:
Yes, as we have to initially fetch it from the Runner. In the initial PR it was returned but I made it now part of the interface. See discussion here: https://github.com/elastic/beats/pull/3439#discussion_r98434613